academic-services/knowledge-exploration-service/concepts-queries.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Natural Language Queries
+description: 
+ms.topic: reference
+ms.date: 2020-02-07
+---
+
+# How MAKES generates semantic interpretations of natural language queries
+
+This article describes how MAKES generates semantic interpretations of a natural language query that enable entities to be retrieved. It is intended for developers using MAKES who require a deeper understanding of the architecture and techniques used.
+
+## Architecture overview and diagram
+
+Processing a natural language query is a multi-stage process that starts with lexical analysis of the query, allowing the query to be rewritten for optimal search performance. The rewritten query is then parsed and interpretations hypothesized using a Context Sensitive Grammar (CSG). Valid hypothesis are resolved into full semantic query expressions which are in turn used to generate the complete interpretation response.
+
+:::image type="content" source="media/concept-semantic-interpretation-flow.png" alt-text="MAKES Semantic Interpretation Generation Architecture":::
+
+## Stage 1: Lexical analysis
+
+:::image type="content" source="media/concept-semantic-interpretation-lexical.png" alt-text="MAKES Semantic Interpretation Generation Architecture - Lexical Analysis":::
+
+Lexical analysis of the user query is done to help ensure that valid interpretation hypothesis can be generated.
```

Comment:
>hypothesis [](start = 84, length = 10)

either "that a valid interpretation hypothesis"
or
"that valid interpretation hypotheses"

Not sure if you are using the plural form here or not.